README.md

```diff
@@ -44,6 +46,15 @@ permitted_domains: 'hashrocket.com|hshrckt.com'
 permitted_emails: 'friend@whitelist.com'
 ```
 
+##### To get admin route working on localhost:
+
+```
+1. Navigate to https://console.developers.google.com/
```

Comment:
@jwworth, possibly this article will be useful: https://developers.google.com/identity/protocols/OAuth2WebServer 
It sounds like a good idea to link to an article maintained by the dependent software.
I think this article covers elements that I have mentioned in the readme. I will let you judge which is more straightforward for on-boarding users. The article in my opinion does great until the section Obtaining OAuth 2.0 access tokens where the location of files needing code changes is lacking. ex. ```#config/application.yml```

Possibly you might consider both the instructions put in the readme and the link as a backup in case a programmer runs into issues.